RequestService/RequestService.Repo/Repository.cs

```diff
@@ -1683,7 +1685,7 @@ public async Task<List<int>> UpdateRequestStatusToCancelledAsync(int requestId,
             {
                 job.JobStatusId = cancelledJobStatus;
                 job.VolunteerUserId = null;
-                AddJobStatus(job.Id, createdByUserID, null, cancelledJobStatus);
+                AddJobStatus(job.Id, createdByUserID, null, JobStatuses.Cancelled, JobStatusChangeReasonCodes.ManualChangeByVolunteer);
```

Comment:
Personally, in scenarios like this, I would prefer to add the JobStatusChangeReasonCode as a parameter, and pass it in from the handler.

This would allow the Repo method to be re-used for automated processes in due course.